.travis.yml

```diff
@@ -24,6 +22,7 @@ script:
 cache:
   directories:
     - node_modules
+    - native/node_modules
```

Comment:
Something i noticed, there is no reason why we cant cache the native `node_modules` too

Reply:
Yep good idea!